src/main/java/com/atlassian/jira/cloud/jenkins/checkgatingstatus/service/GatingStatusRequest.java

```diff
@@ -1,33 +0,0 @@
-package com.atlassian.jira.cloud.jenkins.checkgatingstatus.service;
-
-import jline.internal.Nullable;
-import org.jenkinsci.plugins.workflow.job.WorkflowRun;
-
-import static java.util.Objects.requireNonNull;
-
-public class GatingStatusRequest {
```

Comment:
Removed this file as it conflicted with the `GatingStatusRequest` object I introduced to represent the request to the Jenkins App. 

This type was used in the interface to a service, which was a bit odd, as there was also a `GatingStatusResponse` object, which was used on the client level and not on the service level.

So, I removed this type and just passed the three parameters everywhere where the type was passed before.